types/transaction.go

```diff
@@ -76,7 +77,9 @@ func (t *Transaction) IsContractCreation() bool {
 
 // ComputeHash computes the hash of the transaction
 func (t *Transaction) ComputeHash(blockNumber uint64) *Transaction {
```

Comment:
This function doesn't need blockNumber parameter anymore.

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1904/commits/4ead0caaa91f9ec790fcc4ffe7b574c1d5a5cf19